homeassistant/components/binary_sensor/mychevy.py

```diff
@@ -0,0 +1,68 @@
+"""Support for MyChevy sensors."""
+
+import asyncio
+from logging import getLogger
+
+from homeassistant.components.mychevy import (
+    EVBinarySensorConfig, DOMAIN,
```

Comment:
This sensor belong to the binary sensor domain. So don't overwrite the domain by importing the mychevy domain with the same name. Do:
```py
from homeassistant.components.mychevy import (
    DOMAIN as MYCHEVY_DATA)
```